.gitignore

```diff
@@ -28,4 +28,6 @@ configuration-service/cmd/configuration-service-server/__debug_bin
 
 # IDE files
 .vscode/
+
+*/**/.idea/
```

Comment:
This should not be necessary. Having `.idea/` in our .gitignore should also count for all subfolders (it surely does that for me)